pkg/netutil/netutil.go

```diff
@@ -117,11 +209,8 @@ func (e *CNIEnv) GenerateNetworkConfig(labels []string, id int, name string, plu
 	}, nil
 }
 
-// WriteNetworkConfig writes networkConfig file to cni config path.
-func (e *CNIEnv) WriteNetworkConfig(net *networkConfig) error {
-	if err := os.MkdirAll(e.NetconfPath, 0755); err != nil {
-		return err
-	}
+// writeNetworkConfig writes networkConfig file to cni config path.
+func (e *CNIEnv) writeNetworkConfig(net *networkConfig) error {
```

Comment:
Both the Ctrl-C or No space can cancel the write and maybe there is invalid data in file. Suggest to use atomic write and make sure that content is correct.

Reply:
If needed, maybe another PR